src/backend/parser/parse_func.c

```diff
@@ -1993,6 +1993,11 @@ check_pg_get_expr_arg(ParseState *pstate, Node *arg, int netlevelsup)
 						return true;
 					break;
 
+				case ProcedureRelationId:
+					if (var->varattno == Anum_pg_proc_proargdefaults)
```

Comment:
```suggestion
					if (attnum == Anum_pg_proc_proargdefaults)
```

Would this match the surrounding code better? It's also taken verbatim from the corresponding hunk in REL_9_0_STABLE

Reply:
Yes. Fixed that, thanks!